src/components/shared/formItems/ProjectReconfiguration.tsx

```diff
@@ -0,0 +1,70 @@
+import { t, Trans } from '@lingui/macro'
+import { Form } from 'antd'
+import { CSSProperties, useContext, useState } from 'react'
+import { ThemeContext } from 'contexts/themeContext'
+
+import {
+  ballotStrategies,
+  Strategy,
+} from 'constants/ballotStrategies/ballotStrategies'
+import { getBallotStrategyByAddress } from 'constants/ballotStrategies/getBallotStrategiesByAddress'
+
+import ReconfigurationDrawer from '../ReconfigurationDrawer'
+import ReconfigurationOption from '../ReconfigurationOption'
+
+export default function ProjectReconfiguration({
+  value,
+  onChange,
+  style,
+}: {
+  value: string
+  onChange: (address: string) => void
+  style?: CSSProperties
+}) {
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+  const threeDayDelayStrategy = ballotStrategies()[2]
+
+  const [drawerVisible, setDrawerVisible] = useState<boolean>(false)
+
+  const [selectedStrategy, setSelectedStrategy] = useState<Strategy>(
+    getBallotStrategyByAddress(value),
+  )
+
+  return (
+    <Form.Item
+      name={'reconfiguration'}
+      label={t`Reconfiguration`}
+      style={style ? style : {}}
```

Comment:
This can become `style` if we go with the above suggestion: https://github.com/jbx-protocol/juice-interface/pull/558/files#r810559682

```suggestion
      style={style}
```

Ideally JSX templates contain as little logic as possible :)

Reply:
Sweet ok good tip, I'll fix these up in the tokens form too